src/views/List.jsx

```diff
@@ -1,8 +1,60 @@
 import { ListItem } from '../components';
 import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 export function List({ data }) {
 	const [searchInput, setSearchInput] = useState('');
+	const navigate = useNavigate();
+
+	const WelcomePrompt = () => {
+		return (
+			<>
+				<h3>Shopping list is currently empty...</h3>
+				<button
+					type="button"
+					onClick={() => {
+						navigate('/add-item');
+					}}
+				>
+					Add your first item
+				</button>
+			</>
+		);
+	};
+
+	const FormandList = () => {
```

Comment:
**Nit:** Can the 'A' be capitalized for easier reading? So it would read `FormAndList` instead of `FormandList`